README.md

```diff
@@ -169,3 +169,22 @@ the following while in your virtual enviroment:
 
 Once the above is done you will have autoformatting and pep8 compliance built
 into your workflow. You will be notified of any pep8 errors during commits.
+
+## Copilot Initialisation
+
+Copilot is the deployment of the infrastructure configuration, which is all stored under the copilot folder. The manifest files have been pre-generated by running through various initialisation steps that create the manifest files by prompting a series of questions, but do not _deploy_ the infrastructure.
+
+For each AWS account, these commands will need to be run _once_ to initialise the environment:
```

Comment:
Is this for each environment, or for each app in each environment?

Reply:
For each service in the environment. Essentially there needs to be a _manual_ initialisation step for each of the services, and this will need adding to the README of each service, specialised to each one